source/welcome/keyboard-shortcuts.rst

```diff
@@ -60,6 +60,8 @@ The following navigation keyboard shortcuts are supported only in the Mattermost
     +-------------------------+-----------------+------------------------------------------------------------------------------------+
     | CTRL+SHIFT+TAB          | CMD+SHIFT+TAB   | Navigate to the previous product tab based on your current position.               |
     +-------------------------+-----------------+------------------------------------------------------------------------------------+
+    | CTRL+SHIFT+I            | CMD+SHIFT+I     | Show or hide the channel information pane.                                         |
```

Comment:
Hi @cwarnermm! It works in both. Also I'd like to provide a correction here, for Windows and Linux the shortcut is CTRL+**ALT**+I (not shift)
```suggestion
    | CTRL+ALT+I            | CMD+SHIFT+I     | Show or hide the channel information pane.                                         |
```

![image](https://user-images.githubusercontent.com/785518/171740893-8dbba6e7-5b5d-4185-b734-81cd2c7aace5.png)

Reply:
Thank you, @JulienTant!